src/doc/src/reference/environment-variables.md

```diff
@@ -52,7 +52,7 @@ system:
   detail.
 * `TERM` — If this is set to `dumb`, it disables the progress bar.
 * `BROWSER` — The web browser to execute to open documentation with [`cargo
-  doc`]'s' `--open` flag.
+  doc`]'s' `--open` flag, see [`cargo-doc.browser`] for more details.
```

Comment:
Another instance that needs to be renamed.

Reply:
Fixed, mentioned it also in the manual pages for the commands.